OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -18,7 +18,7 @@ public class BuildableInfo : TraitInfo<Buildable>
 			"This can be prefixed with ! to invert the prerequisite (disabling production if the prerequisite is available)",
 			"and/or ~ to hide the actor from the production palette if the prerequisite is not available.",
 			"Prerequisites are granted by actors with the Building trait (with a prerequisite string given by the lower case actor name)",
```

Comment:
> "Prerequisites are granted by actors with the Building trait (with a prerequisite string given by the lower case actor name)"

This isn't true anymore after this change. `ProvidesPrerequisite` will be the only source for prerequisites.